lib/PuppeteerSharp/BrowserData/Firefox.cs

```diff
@@ -90,7 +90,7 @@ private static string GetArchive(Platform platform, string buildId)
 
         private static Dictionary<string, object> GetDefaultPreferences(Dictionary<string, object> preferences)
         {
-            var server = "dummy.test";
+            const string server = "dummy.test";
```

Comment:
This enables constant string interpolation.

![image](https://github.com/hardkoded/puppeteer-sharp/assets/919634/dc8de07f-4220-42bd-97e0-01774e92e975)
[SharpLab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA+ABAjABgAQByEAdgMIkDOALgBQCUAsAFADezeHekx1e2+FGFABuQvAF48AIgAmAVwC2CgJ4A6KjGpSA3O04YA7HgAkUgBZUqABxAB6Wy0EihAX1vyly22ZgBDADZUZrBW0FS2OswuzMz8eOQ8tIysehxxTqJQEtIeKuqaVJFMnHxGphbWdg4Zru6KKt5+gcEwoVDhRS5AA=)